src/wallet/wallet.cpp

```diff
@@ -4374,6 +4374,15 @@ bool CWallet::SetActiveHDKey(const CExtPubKey& xpub)
     return WalletBatch(GetDatabase()).WriteActiveHDKey(xpub);
 }
 
+std::optional<CExtPubKey> CWallet::GetActiveHDPubKey()
```

Comment:
in e693bd02610df20f73c008b03c739da5d69fcf76

nit: inconsistent naming.

Propose to rename to `GetActiveHDKey()` analogous to `LoadActiveHDKey()` and `SetActiveHDKey()`

Reply:
Done as suggested